fulp_modules/features/species/beefmen/phobetor_trauma.dm

```diff
@@ -6,10 +6,10 @@
 
 /datum/brain_trauma/special/bluespace_prophet/phobetor
 	name = "Sleepless Dreamer"
-	desc = "The patient, after undergoing untold psychological hardship, believes they can travel between the dreamscapes of this dimension."
+	desc = "The patient (having undergone untold psychological hardship,) believes they can travel between metaphysical dreamscapes."
```

Comment:
I think the parenthesis makes it worse here, i think there isn't supposed to have a comma, either.

Reply:
I do believe this works as a parenthetical— the sentence is mostly the same with it as it is without it.

As for the comma: a brief search around the internet has revealed to me that there is no "proper" place to put a comma near a parenthetical. It's almost entirely a matter of convention; until we adopt a standardized writing style, anything could be correct.

If you'd prefer it I could just set the nonessential portion of the sentence aside with commas.

Reply:
I can't find anywhere of anything saying there should have commas in parentheses, where are you getting this from?

Reply:
I have no idea. I admit to being in the wrong here, but I'm honestly rather surprised. At some point in my life I was of the belief that commas _should_ go after parentheses, but then I was evidently taught to the contrary. To put it dramatically: I almost feel like I'm suffering from the Mandela Effect in one way or another— as if I were spontaneously transported into a different reality where commas go outside of parentheses instead of inside them. There's almost certainly a more reasonable explanation, but in the meantime all I can say is "waow".

I'll fix this.